src/examples/LottieExtension.cpp

```diff
@@ -85,6 +85,11 @@ void tvgDrawCmds(tvg::Canvas* canvas)
         cout << "Failed to override the slot" << endl;
     }
 
+    //Revert slot
```

Comment:
Rather than example, I think it's better to include this minor showcase in the unit tests.